lib/main/widgets/pet_list_item.dart

```diff
@@ -0,0 +1,105 @@
+      child: Row(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              border: Border.all(color: const Color.fromRGBO(196, 196, 196, 1)),
+              color: Colors.grey,
+            ),
+            width: 115.0,
+            height: 92.0,
+          ),
+          Container(
+            decoration: BoxDecoration(
+                border:
+                    Border.all(color: const Color.fromRGBO(196, 196, 196, 1)),
+                color: Colors.white),
+            width: 221.0,
+            height: 92.0,
+            child: Padding(
+              padding:
+                  const EdgeInsets.symmetric(vertical: 8.0, horizontal: 9.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Row(
+                        children: [
+                          const Icon(
+                            Icons.folder_shared_outlined,
+                            size: 28.0,
+                            color: Colors.black,
+                          ),
+                          Text(
+                            namePet,
+                            style: GoogleFonts.montserrat(
+                              fontSize: 25.0,
+                              fontWeight: FontWeight.w500,
+                              color: Colors.black,
+                            ),
+                          )
+                        ],
+                      ),
+                      Row(
+                        children: [
+                          const Icon(
+                            Icons.pets,
+                            size: 28.0,
+                            color: Colors.black,
+                          ),
+                          Text(
+                            breedPet,
+                            style: GoogleFonts.montserrat(
+                              fontSize: 15.0,
+                              fontWeight: FontWeight.w600,
+                              color: const Color.fromRGBO(0, 0, 0, 0.7),
+                            ),
+                          )
+                        ],
+                      ),
+                    ],
+                  ),
+                  Column(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: const [
+                      Icon(
+                        Icons.edit,
+                        size: 28.0,
+                        color: Colors.black,
+                      ),
+                      Icon(
+                        Icons.restore_from_trash_outlined,
+                        size: 28.0,
+                        color: Colors.black,
+                      ),
```

Comment:
Should we make this into buttons? (You should use something like IconButton or Inkwell)